index.html

```diff
@@ -252,7 +252,7 @@ <h1><%= @article.title %></h1>
 
         <div class="example__content common-content">
           <h3>Action Views mix Ruby and HTML.</h3>
-          <p>Templates can use the full version of Ruby, excessive code is extracted into helpers, and the domain model is used directly and inter-vowed with the HTML.</p>
+          <p>Templates can use the full versatility of Ruby, excessive code is extracted into helpers, and the domain model is used directly and inter-vowed with the HTML.</p>
```

Comment:
"inter-vowed" seems... interesting too 😅 

@dhh should that be inter-woven?

Reply:
lol yes